cwel/component/mini-contact/mini-contact.html

```diff
@@ -0,0 +1,98 @@
+{% macro default(title, title2, text) %}
+    <section class="container--dark">
+        <div class="container">
+            <div class="mini-contact">
+                <div class="mini-contact-grid">
+                    <h4 class="mini-contact__title">{[ title ]}</h4>
+                    <p class="mini-contact__text">
+                        {[ text ]}
+                    </p>
+                </div>
+                <div class="mini-contact-grid">
+                    <h4 class="mini-contact__title">{[ title2 ]}</h4>
+                    <ul class="mini-contact__text">
```

Comment:
@ambtlv I'm thinking the contents of `.mini-contact-grid` would be CMSable. Might want to omit the class names for elements inside.